tests/test_message/test_message_create.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from .event_sample import request, headers
+
+
+@pytest.mark.parametrize(
+    "data, headers, code",
+    [
+        pytest.param(
+            request,
+            headers,
+            200,
+            id="MESSAGE Endpoint - 200 Success",
+        ),
+        pytest.param(
+            request,
+            {**headers, "X-Hub-Signature-256": ""},
+            400,
+            id="MESSAGE Endpoint - 400 Bad Request"
+        ),
+        pytest.param(
+            None,
+            headers,
+            415,
+            id="MESSAGE Endpoint - 415 Unsupported Media Type"
+        )
+    ]
+)
+
+@pytest.mark.usefixtures("create_service")
+def test_message_create(client, data, headers, code):
+    response = client.post("/message/1", json=data, headers=headers)
+    assert response.status_code == code
+
+
+@pytest.mark.parametrize(
+    "data, headers, code",
+    [
+        pytest.param(
+            request,
+            headers,
+            404,
+            id="MESSAGE Endpoint - Not Found",
+        )
+    ]
+)
+
+@pytest.mark.usefixtures("create_service")
+def test_not_found(client, data, headers, code):
+    response = client.post("/message/2", json=data, headers=headers)
+    assert response.status_code == code
```

Comment:
I suspect that these can be combined to be four parameters under one testing function. Simply parameterize the URL path too (as in `f"/message/{id}"`, where the `id` is provided from the parameters) and you should be fine.